examples/solvers/pdhg_tomography_tgv.py

```diff
@@ -89,7 +89,7 @@
     E * odl.ComponentProjection(domain, 1))
 
 # Do not use the g functional, set it to zero.
-g = odl.solvers.ZeroFunctional(domain)
+f = odl.solvers.ZeroFunctional(domain)
```

Comment:
Update comment above

Reply:
The same problem appears in two other places.
* https://github.com/odlgroup/odl/blob/c8ab25dcffcee8010a65202af6a1a3f7bef84329/examples/solvers/pdhg_denoising_tgv.py#L88
* https://github.com/odlgroup/odl/blob/c8ab25dcffcee8010a65202af6a1a3f7bef84329/examples/solvers/pdhg_tomography.py#L51